service/application/src/main/java/com/solace/maas/ep/event/management/agent/EMAApplication.java

```diff
@@ -51,6 +51,8 @@ public static Properties getDefault() {
 
         properties.put("spring.application.name", "event-management-agent");
         properties.put("server.port", "8180");
+        properties.put("management.statsd.metrics.export.enabled", "false");
```

Comment:
How did 007 squad use this feature btw considering that ep-core haven't been updated to generate this config? i.e. the config we send in sidekick request probably wouldn't have this value set at all!

Reply:
we talked offline.